### PR TITLE
Fix blackbar when hitting back on device add

### PR DESCRIPTION
**Pull request by chrisnojima**
this fixes clicking back when adding a new device. 
Core sends us back 2 possible error values when we cancel so we need to ignore both. previously we ignored one, then switched to the other, now we handle both

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

File: shared/actions/__tests__/provision.test.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ describe('provisioningManagerProvisioning', () => {
       expect(response.result).not.toHaveBeenCalled()
       expect(response.error).toHaveBeenCalledWith({
         code: RPCTypes.constantsStatusCode.scinputcanceled,
-        desc: Constants.cancelDesc,
+        desc: 'Input canceled',
```
Comment (buoyad) [inline shared/actions/__tests__/provision.test.js]: Can we keep this as a constant?

Comment (chrisnojima) [reply shared/actions/__tests__/provision.test.js]: i think its ok for this test to know about the value. i don't want to export the individual constants out

Comment (buoyad) [reply shared/actions/__tests__/provision.test.js]: Oh I misread this as the main action file. Sounds good

**Pull request merged**